instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -495,42 +533,56 @@ def __init__(
         tracer_provider=None,
         meter_provider=None,
     ):
+        # initialize semantic conventions opt-in if needed
+        _OpenTelemetrySemanticConventionStability._initialize()
```

Comment:
This class-based approach requires the instrumentations to worry about calling for the initialization themselves, that should not be the case, `from opentelemetry.x.y.z.semconv import http_stability_mode` should take care of getting the HTTP stability mode by itself.